cornice/resource.py

```diff
@@ -71,6 +71,14 @@ class User(object):
     else:
         prefixes = ('',)
 
+    if 'collection_pyramid_route' in kw:
+        if kw['collection_pyramid_route'] == kw['pyramid_route']:
+            msg = "Warning: collection_pyramid_route and " \
+                  "pyramid_route are not distinct."
+            warnings.warn(msg)
+
+        prefixes = ('', 'collection_')
+
```

Comment:
What happens when `*path` and `*route` are specified?

Reply:
Fixed with test and exception